src/modules/esl-share/core/esl-share-action.ts

```diff
@@ -17,16 +19,19 @@ export abstract class ESLShareBaseAction {
     ESLShareActionRegistry.instance.register(action);
   }
 
+  /** Checks if this action is available on the user's device */
   public get isAvailable(): boolean {
     return true;
   }
 
+  /** Returns ShareData object for button  */
```

Comment:
```suggestion
  /** @returns {@link ShareData} object for button  */
```
BTW: we need to consider the prefix `ESL` for types